string class_name
int32 u_min
int32 u_max
int32 v_min
int32 v_max
geometry_msgs/Point center
float32 width
float32 height
float32 depth
string  rle_encoded_mask
float32 score